packages/expo-dev-menu/react-native.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  dependency: {
+    platforms: {
+      ios: {
+        /**
+         * We need this property to trick `react-native-cli`. Normally, this tool looks for the XCode project.
+         * However, this package is a library so we don't have a project file.
```

Comment:
This is actually not true — library may have its `.xcodeproj` (and that's the requirement for autolinking) but in our repo we don't use them because it would be a big maintenance cost to keep them up-to-date. We could generate them using XcodeGen (I used it for prebuilds), but I don't really see any reason why to keep them in the repo since everything we need is in xcworkspaces.

By the way, expo-random package which has a dummy xcodeproj just to make it autolinkable 😄 but in my opinion this thing in the config is better.